workers-docs/src/content/tutorials/build-a-serverless-function/_index.md

```diff
@@ -131,7 +131,7 @@ By default, the QR code is generated as a PNG. Construct a new instance of `Resp
 ```javascript
 const generate = async request => {
   // ...
-  return new Response(qr_png, { headers })
+  return new Response(qr_png, { headers: { 'Content-Type': 'image/png' } })
```

Comment:
ah, i realized this actually should be fixed by providing that headers var vs inlining it, which will work w/ the rest of the examples:

```suggestion
  const headers = { 'Content-Type': 'image/png' }
  return new Response(qr_png, { headers })
```